src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -31,6 +31,7 @@ namespace Utils
         {
             rocksdb::Options options;
             options.create_if_missing = true;
+            options.compression = rocksdb::kLZ4HCCompression;
```

Comment:
I've found that `LZ4` is always recommended over the default Snappy
https://github.com/facebook/rocksdb/wiki/Compression

But are we sure that `LZ4HC` is better than `LZ4`?
Here we have some benchmarks 

http://www.lmdb.tech/bench/inmem/compress/RocksDB/

Reply:
I made the test with the three algorithms and wrote in the PR summary. 
I agree with the web page, `zlib` is the best option but is very slow. `zl4hc` has good performance and time rather than `lz4`

Reply:
Great, thank you!